Frends.Salesforce.ExecuteQuery/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.1] - 2022-05-27
+### Added

```

Comment:
This should state `Changed`, since you haven't added any new functionality, but fixed existing one.